interface/src/app/analyze/analyze.js

```diff
@@ -58,9 +54,7 @@ angular.module('adage.analyze', [
       },
 
       scroll_to_id: function(id) {
-        $log.info('scroll_to_id called with: ' + id);
```

Comment:
Ah... but I'm noticing that I removed the `$anchorScroll` entirely but it's still getting injected above. I'll fix that too.